File: OneDrive/Desktop/Algorithm/TransposeMatrix/src/TransposedMatrix.js
function transposedMatrix(matrix) {
    let transposed = [matrix[0].length][matrix.length];
    for (let row = 0; row < matrix.length; row++) {
        for(let column = 0; column < matrix[row].length; column++) {
            if (!transposed[column]) {
                transposed[column] = [];
            }
            transposed[column][row] = matrix[row][column];
        }
    }
    return transposed;
}


let matrix = [[1, 2, 3], [4, 5, 6], [7, 8, 9]];
console.log(transposedMatrix(matrix));

File: OneDrive/Desktop/Algorithm/TransposeMatrix/src/TransposedMatrix.java
import java.util.Arrays;

public class TransposedMatrix {
    public int[][] transposeMatrix(int[][] integers){

        int[][] transposed = new int[integers[0].length][integers.length];
        for(int row = 0; row < integers.length; row++){

            for(int column = 0; column < integers[row].length; column++){

                transposed[column][row] = integers[row][column];
            }
        }
        System.out.println(Arrays.deepToString(transposed));
        return transposed;
    }
}
